app/assets/src/components/views/bulk_download/ChooseStep.jsx

```diff
@@ -178,6 +203,20 @@ class ChooseStep extends React.Component {
     // Set different props for the dropdown depending on the field type.
     switch (field.type) {
       case "file_format":
+        if (
+          downloadType.type === READS_NON_HOST_DOWNLOAD_TYPE &&
```

Comment:
extract this pattern to a methods such as `isTaxonWithReads`

Reply:
`isReadsNonHostDownloadType(downloadType)` barely feels shorter. This particular case doesn't feel worth it to me.